### DISCUSSION TITLE
1.x.antdv.com is down

### DISCUSSION THREAD

**Discussion by jysaa5** [vueComponent/ant-design-vue#6125] [Q&A]
I still use ant-design-vue 1.7.8 version.
so I need ant-design-vue 1.x documents.. 

https://1x.antdv.com/ is not working. 
what's problem? 

1.x version is end ?

**Comment by ukqfhf** [comment]
The link is working for me

**Reply by jysaa5** [reply]
Right. 
But it didn't work for several hours. 
So I couldn't get it to work.

Thank you for reply~
